packages/geo/src/lib/print/shared/print.service.ts

```diff
@@ -536,10 +587,20 @@ export class PrintService {
       // If a title need to be added to canvas
       if (title !== '') {
         // Set font for title
+        // Ajuster en fonction de la longueur du titre
+
+        newContext.font = '26px Calibri';
+        positionHCanvas = 30;
+        newContext.textAlign = 'center';
+        newContext.fillText(width, width / 2, 20, width * 0.9);
+      }
+      if (subtitle !== '') {
+        // Set font for title
+        // Ajuster en fonction de la longueur du titre
         newContext.font = '26px Calibri';
         positionHCanvas = 30;
         newContext.textAlign = 'center';
-        newContext.fillText(title, width / 2, 20);
+        newContext.fillText(width, width / 2, 20, width * 0.9);
```

Comment:
Why width as the first parameter ?
The syntax of the function is:
void ctx.fillText(texte, x, y [, largeurMax]);